README.adoc

```diff
@@ -275,6 +279,34 @@ Below you can find an example of a Logback configuration (file named `https://gi
 NOTE: If you're using a custom `logback-spring.xml` then you have to pass the `spring.application.name` in
 `bootstrap` instead of `application` property file. Otherwise your custom logback file won't read the property properly.
 
+===== Propagating Span Context
+
+The span context is the state that must get propagated to any child Spans across process boundaries.
+Part of the Span Context is the Baggage.
+
+Baggage is a set of key:value pairs stored in the span context. Baggage travels together with the trace
+and is attached to every span. Spring Cloud Sleuth will understand that a header is baggage related if the HTTP
+ header is prefixed with `baggage-` and for messaging it starts with `baggage_`.
+
+IMPORTANT: Since you can attach as many baggage elements as you want and it's propagated through the network
+then if it's weighing too much or you have too many elements then you it may decrease system throughput or increase RPC latencies.
+
+Example of setting span:
+
+[source,java]
+----
+Unresolved directive in intro.adoc - include::http://raw.github.com/spring-cloud/spring-cloud-sleuth/master/spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/multiple/MultipleHopsIntegrationTests.java[tags=baggage,indent=0]
+----
+
+====== Baggage vs. Span Tags
+
+Baggage travels with the trace (i.e. every child span contains the baggage of its parent). Zipkin has no knowledge of
+baggage and will not even receive that information.
+
+Tags are not passed from a parent to a child, however they are sent to Zipkin and will get presented in the
+Zipkin UI for every span.
```

Comment:
tags are attached to a specific span, so they aren't presented "for every span", rather the span they are recorded in. that said you can search by tag...